src/libraries/System.IO.Pipes/ref/System.IO.Pipes.cs

```diff
@@ -95,6 +95,7 @@ public enum PipeOptions
         None = 0,
         CurrentUserOnly = 536870912,
         Asynchronous = 1073741824,
+        FirstPipeInstance = 524288
```

Comment:
Can you please add back the attribute and suppress the analyzer _in source_ where needed.

Reply:
Done